src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -146,6 +148,7 @@ internal void AddProvider(string workerDir)
                             CountOptions = workerProcessCount,
                         };
                         _workerDescriptionDictionary[workerDescription.Language] = rpcWorkerConfig;
+                        ReadLanguageWorkerFile(arguments.WorkerPath);
```

Comment:
also any reason to do this inside this class? seems like this would be better suited in one of the warmup middlewares where we already call prejit.

Reply:
You are right - reading files part for Placeholder / Host Warmup middlewares is more accurate to ensure files continue to stay in-memory. As of now they do not have access Rpc Worker Configs.   I will take a look if this can be achieved without too many changes

Reply:
@balag0 - Scoping the changes in the PR  - lets revisit if the code needs to be moved to middleware to keep worker files in memory